bin/functions/filesystem/downloadFile.mjs

```diff
@@ -35,7 +35,7 @@ export async function downloadFile(url, folder, file) {
     const _file = fs.createWriteStream(PATH);
     if (!file.toLowerCase().includes('keepcols') && file.includes('.svg')) {
```

Comment:
could we add keepcols to the name and don't do this at all then or would that not be sufficient?

Reply:
i don't think it would, as that would affect ~300 icons, which should all get the currrentColor

we could reduce the complexity of the regex to `((?<!url\()#\w+)`, so that it just checks if the text doesn't start with `url(`, but everything else stays the same

Reply:
or maybe just do a 

```ts
const regex = new RegExp('url\((\w+)\)',gi);
if (!file.toLowerCase().includes('keepcols') && file.includes('.svg') && !regex.test(file.toLowerCase())) {
...
}
```
or am I missing something here?